src/wp-includes/feed-rss2-comments.php

```diff
@@ -32,33 +32,41 @@
 	?>
 >
 <channel>
-	<title><?php
-		if ( is_singular() ) {
-			/* translators: Comments feed title. 1: Post title */
-			printf( ent2ncr( __( 'Comments on: %s' ) ), get_the_title_rss() );
-		} elseif ( is_search() ) {
-			/* translators: Comments feed title. 1: Site name, 2: Search query */
-			printf( ent2ncr( __( 'Comments for %1$s searching on %2$s' ) ), get_bloginfo_rss( 'name' ), get_search_query() );
-		} else {
-			/* translators: Comments feed title. 1: Site name */
-			printf( ent2ncr( __( 'Comments for %s' ) ), get_wp_title_rss() );
-		}
-	?></title>
+	<title>
+	<?php
+	if ( is_singular() ) {
+		/* translators: Comments feed title. 1: Post title */
+		printf( ent2ncr( __( 'Comments on: %s' ) ), get_the_title_rss() );
+	} elseif ( is_search() ) {
+		/* translators: Comments feed title. 1: Site name, 2: Search query */
+		printf( ent2ncr( __( 'Comments for %1$s searching on %2$s' ) ), get_bloginfo_rss( 'name' ), get_search_query() );
+	} else {
+		/* translators: Comments feed title. 1: Site name */
+		printf( ent2ncr( __( 'Comments for %s' ) ), get_wp_title_rss() );
+	}
+	?>
+	</title>
 	<atom:link href="<?php self_link(); ?>" rel="self" type="application/rss+xml" />
-	<link><?php (is_single()) ? the_permalink_rss() : bloginfo_rss("url") ?></link>
-	<description><?php bloginfo_rss("description") ?></description>
-	<lastBuildDate><?php
+	<link><?php ( is_single() ) ? the_permalink_rss() : bloginfo_rss( 'url' ); ?></link>
+	<description><?php bloginfo_rss( 'description' ); ?></description>
+	<lastBuildDate>
+	<?php
```

Comment:
In this case of putting the `<?php` tag on its own line, the output is larger than before. In the other lines with single line of code for output, the php tag is not forced to its own line. 
The XML can handle the extra white space, but should the code standard force that?

Reply:
I think this rule applies when there are multiple lines within the `<?php ?>` tags

Reply:
Yes, that's what the standard says, but are we blindly following the WP standard? 
In the case of white space in `href`, we backported their fix. I'm just saying that the type of output being generated might not handle the extra white space that the standard is enforcing. 
I suppose it's okay in XML such as this is, but I haven't looked up the XML rules.

Reply:
I read something about XML parsers converting all white space to single spaces and then trimming first and last, so it probably is not a problem.

Reply:
We can probably leave or the other option is to consider a backport of:
https://core.trac.wordpress.org/changeset/45247

Reply:
I'm okay either way, but that changeset does not look independent, so it probably should not be part of a layout PR.

Reply:
The output is something completely different. We might need to add a commit fixing the whitespace at start and end. Add an ignore phpcs rule if needed.